### PR TITLE
templates: Added template for JaguarDB 

### PR DESCRIPTION
**Pull request by DannyMang**
- **Description:**: added langchain template for JaguarDB

### CODE REVIEW

**Comment by DannyMang** [comment]
For when someone looks at my changes, would it possible for me to receive feedback? ( as this is my first PR towards langchain, thank you!)

File: templates/rag-jaguardb/rag_jaguardb/chain.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+
+url = "http://192.168.3.88:8080/fwww/"
+pod = "vdb"
+store = "langchain_test_store"
+vector_index = "v"
+vector_type = "cosine_fraction_float"
+vector_dimension = 1536
+embeddings = OpenAIEmbeddings()
+vectorstore = jaguar(
```
Comment (efriis) [inline templates/rag-jaguardb/rag_jaguardb/chain.py:25]: does this work for you? I think this has to be the Jaguar class exported from that file

File: templates/rag-jaguardb/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+
+# rag-jaguardb
+
+This template performs RAG using JaguarDB and OpenAI.
+
+## Environment Setup
+
+You should export two environment variables, one being your Jaguar URI, the other being your OpenAI API KEY.
+If you do not have a JaguarDB URI, see the `Setup Jaguar` section at the bottom for instructions on how to do so.
```
Comment (efriis) [inline templates/rag-jaguardb/README.md:9]: I believe this section is missing

**Review by efriis — COMMENTED** [review]
Two comments, and would be great to get lint/CI passing!

**Comment by DannyMang** [comment]
@efriis uploaded commits that resolved those two mentioned issues! Also is it possible to do lint testing on my IDE without having to push a commit to GitHub?

**Review by efriis — APPROVED** [review]

**Pull request merged**